### ISSUE TITLE
Rendering on Operations in multiple datasets (spanish)

### ISSUE THREAD

**Issue by gallardoalba** [galaxyproject/galaxy#12822]
**Describe the bug**
When using the Spanish version of Galaxy, operations in multiple datasets don't render well (two rows), and the operations cannot be read unless the menu size is expanded.

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: 21.09

**To Reproduce**
1. Select Spanish as the default language.
2. Click on "Operaciones en conjuntos de datos múltiples" (operations in multiple datasets)
3. See the result

**Expected behavior**
Options in one row and text of operations visible.

**Screenshots**

![Screenshot from 2021-11-01 11-43-15](https://user-images.githubusercontent.com/26678117/139659728-07bbba55-95d4-4361-af64-fbd074406f8d.png)

**Comment by mvdbeek** [comment]
Does the beta history have the same problem ?

**Comment by gallardoalba** [comment]
> Does the beta history have the same problem ?

Beta history seems to not be translated to Spanish at all.

**Comment by hexylena** [comment]
We are teaching a spanish language Galaxy course in <3 weeks, if there's any hot patch for this we could apply it would be really helpful

fyi @nomadscientist @shiltemann

**Comment by dannon** [comment]
I will take a look at the popup being behind the center panel now.  For the extra big button there -- I don't suppose there's a slightly more concise translation we can use as an easy fix?

**Comment by hexylena** [comment]
maybe @galaxyproject/training-spanish have ideas there?

**Comment by davelopez** [comment]
Maybe `Para los seleccionados...` the 'todos' (all) can be implicit.

**Comment by hexylena** [comment]
yeah I guess it's really due to conjuntos de datos being repeated every time

**Comment by davelopez** [comment]
Oh, I see, I was focusing only on the button wrapping. Then I would say `datasets` instead of `conjuntos de datos` may be OK... I mean, I guess someone slightly familiar with science and data analysis should understand that but otherwise I'm out of options here.

**Comment by dannon** [comment]
No worries on the text contents of the menu -- that's fixed in #12906.  Still not sure what to do about the buttons themselves though -- I did try @davelopez suggestion there but it's not quite enough.

**Comment by mariipia10** [comment]
I'd agree with @davelopez but just in case if `conjuntos de datos ` is replaced by `datasets` I'd put somewhere the headsup for the translation. 
@dannon nice job! I wonder is there anyway that we can detect if the user is using the spanish version and then act consequently?
